### PR TITLE
docs: improve cookies

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
See https://forum.codeigniter.com/showthread.php?tid=88176

- fix docs
- refactor

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/libraries/cookies/017.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+use CodeIgniter\Cookie\Cookie;
+use Config\Services;
+use DateTime;
```
Comment (datamweb) [inline user_guide_src/source/libraries/cookies/017.php:5]: CI4 has a class `use CodeIgniter\I18n\Time;`. These documents are codeigniter, I think it is better to use the internal library.
Just a personal preference.

Comment (kenjis) [reply user_guide_src/source/libraries/cookies/017.php]: Changed to use undocumented `max-age`.

**Review by datamweb — APPROVED** [review]

**Review by MGatner — APPROVED** [review]
Maybe all our config classes should include "Config" in the name? 🤔

**Comment by kenjis** [comment]
> Maybe all our config classes should include "Config" in the name? 🤔

It may be better.

**Comment by MGatner** [comment]
Something to consider for v5. I frequently have to alias my imports so I can use both `Widget` and `Widget as WidgetConfig`. It also feels weird to typehint things `App` that just use the object properties.

**Pull request merged**